### DISCUSSION TITLE
japa automated tests

### DISCUSSION THREAD

**Discussion by lucasabrantes1** [adonisjs/core#3209] [Help]
Good morning everyone, I'm trying to perform tests and through a custom script by nodemon, when I try to run the command I created japa:watch, it manages to run the test but right after the error below, I tried to remove the folder node modules and install again, but none of that worked.
![image](https://user-images.githubusercontent.com/62405928/135088712-fecbbfa3-5180-4c6d-be49-aef5b78d8dea.png)

**Comment by WeyIin** [comment]
So.. you can't remove node modules at all? What does it say?
